modules/cloud-config-container/simple-nva/variables.tf

```diff
@@ -14,6 +14,18 @@
  * limitations under the License.
  */
 
+variable "bgp_config" {
+  description = "BGP configuration for FR Routing container running on the NVA."
+  type = object({
+    daemons    = optional(string)
+    enable     = optional(bool)
```

Comment:
`enable     = optional(bool, false)` or just ignore as `null` will be boolean false and works too

Reply:
you can then use `default = {}` and and it's safer to also set `nullable = false`

Reply:
thanks for the hint! Done